From 43dade2e393f64e04259449b7ef3aa2cca489673 Mon Sep 17 00:00:00 2001
From: Suneel Garapati <sgarapati@caviumnetworks.com>
Date: Wed, 2 Aug 2017 13:18:15 -0700
Subject: [PATCH 0680/1239] mach: thunderx: remove call to dram size

remove call to dram size as atf call is used in board file.
keep dummy entry for lowlevel init to avoid generic call.

Signed-off-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 arch/arm/mach-thunderx/lowlevel_init.S | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/arch/arm/mach-thunderx/lowlevel_init.S b/arch/arm/mach-thunderx/lowlevel_init.S
index 2933ece17c..3e59f595ef 100644
--- a/arch/arm/mach-thunderx/lowlevel_init.S
+++ b/arch/arm/mach-thunderx/lowlevel_init.S
@@ -29,9 +29,7 @@ save_boot_params:
 ENTRY(lowlevel_init)
 	mov	x29, lr			/* Save LR */
 
-	ldr	x0, =THUNDERX_DRAM_SIZE
-	mov	x1, xzr
-	smc	#0			/* Get the DRAM size */
+	/* any lowlevel init should go here */
 
 	mov	lr, x29			/* Restore LR */
 	ret
-- 
2.29.0

